pkg/render/compliance.go

```diff
@@ -277,12 +273,12 @@ var (
 	complianceReplicas int32 = 1
 )
 
-const complianceServerPort = 5443
+const ComplianceServerPort = 5443
```

Comment:
Does this need to be public now? I can't see where it's used outside of this package (but might have missed something)